src/parallel/MPI_TIMED/ProcessTimer.h

```diff
@@ -0,0 +1,111 @@
+/*
+ * ProcessTimer.h
+ *
+ *  Created on: Apr 12, 2020
+ *      Author: jeremyharisch
+ */
+
+#ifndef MARDYN_PROCESSTIMER_H
+#define MARDYN_PROCESSTIMER_H
+
+#include <array>
+#include <fstream>
+#include <map>
+#include <vector>
+
+#include <mpi.h>
+
+class ProcessTimer {
+public:
+	static ProcessTimer& get() {
+		static ProcessTimer processTimer;
+		return processTimer;
+	}
+
+	/*! @brief Switches between the profiling Levels
+	 *  This Fnction should be called right before and after a corresponding MPI-Function-Call
+	 *  to exclude only this one
+	 *
+	 *  @param level ==1 Profiling is enabled(default); ==0 Profiling is disabled
+	 */
```

Comment:
- missing documentation of the return value
- A bit weird that this uses a different block comment style than the rest of the file... but I guess that's just ls1 🤷